geodiff/src/drivers/postgresdriver.cpp

```diff
@@ -1005,9 +1023,16 @@ void PostgresDriver::dumpData( ChangesetWriter &writer, bool useModified )
   if ( !mConn )
     throw GeoDiffException( "Not connected to a database" );
 
+  const std::vector<std::string> tablesToSkip = context()->tablesToSkip();
   std::vector<std::string> tables = listTables();
   for ( const std::string &tableName : tables )
   {
+    // skip table if necessary
+    if ( std::any_of( tablesToSkip.begin(), tablesToSkip.end(), std::bind( std::equal_to< std::string >(), std::placeholders::_1, tableName ) ) )
```

Comment:
What about adding a function to the context that would check whether a table should be skipped? e.g. `bool isTableSkipped( const std::string &tableName )` - that would hide the implementation detail away from the other code (and if we even choose to use some other data structure to keep "tables to skip" information (e.g. a set), it would be easier to change)